examples/aurora-mysql/main.tf

```diff
@@ -97,3 +71,43 @@ module "rds_cluster" {
   policy_type            = "TargetTrackingScaling"
   predefined_metric_type = "RDSReaderAverageCPUUtilization"
 }
+
+resource "aws_backup_vault" "this" {
+  name          = "${local.cluster_name}-backup-vault"
+  force_destroy = true
+  kms_key_arn   = data.aws_kms_key.supporting.arn
+  tags          = local.tags
+}
+
+resource "aws_backup_plan" "this" {
+  name = "${local.cluster_name}-backup-plan"
+  rule {
+    rule_name         = "${local.cluster_name}-backup-rule"
+    target_vault_name = aws_backup_vault.this.name
+    schedule          = "cron(0 12 * * ? *)"
+
+    lifecycle {
+      delete_after = 14
+    }
+  }
+}
+
+resource "aws_backup_selection" "this" {
+  iam_role_arn = aws_iam_role.backup.arn
+  name         = "${local.cluster_name}-backup-selection"
+  plan_id      = aws_backup_plan.this.id
+
+  resources = [
+    module.rds_cluster.arn
+  ]
```

Comment:
What do you think about having these features as part of the module? Although may need to be in another sprint

Reply:
Yes, this is a viable option, with an option to use an external backup resource/module as well